src/components/detail/ContentSidebar.vue

```diff
@@ -5,10 +5,10 @@
       <div v-for="(item, index) in toc"
         :key="index"
         class="toc--item"
-        @click="scrollToHeading(item.text, item.depth)"
+        @click="scrollToHeading(item.id)"
       >
         <mdi-menu-right class="mdi-circle" />
-        {{ item.text }}
+        {{ item.actualText}}
```

Comment:
`text` should be fine. Or is there any other use of `.text`?
```suggestion
        {{ item.text}}
```